core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -284,31 +284,36 @@ public SaslConnectionParams getSaslParams() {
     return saslSupplier.get();
   }
 
-  public synchronized BatchWriterConfig getBatchWriterConfig() {
+  public synchronized static BatchWriterConfig getBatchWriterConfig(Properties props) {
```

Comment:
Synchronized should go with the instance method and not the static method.

Reply:
Updated